lib/matplotlib/texmanager.py

```diff
@@ -81,8 +89,20 @@ def __new__(cls):
 
     def get_font_config(self):
         ff = rcParams['font.family']
-        if len(ff) == 1 and ff[0].lower() in self._font_families:
-            self._font_family = ff[0].lower()
+        ff_val = ff[0].lower()
+        _aux_check = False
```

Comment:
perhaps _aux_check can be renamed to something more descriptive?

Reply:
I changed it to `reduced_notation` but I still think that it could have a better name. I only couldn't think any.